docs/techpass-status.md

```diff
@@ -18,6 +18,12 @@ No upcoming scheduled maintenance activity.
 
 ## Previous incidents
 
+| Date | 02 July 2024 |
+|------|--------------|
+| **Issue summary** | **Resolved**<br><br>*Updated on: 3 July 2024, 16:40 SGT*<br><br>The issue has been resolved and verified. Going forward, profiles will be created with the required attributes. Users should no longer experience issues accessing the applications.<br><br>*Updated on: 3 July 2024, 16:40 SGT* We detected some issues with our account creation service. The team investigated and assessed the impact. We will update everyone as soon as we have more information and notify you of any impacts.<br><br>**Workaround**<br> 50 users affected received an email and were able to access TechPass. With this workaround, users received the invitation emails, but there may have been a delay in delivery.<br><br>*Posted on: 02 July 2024, 16:00 SGT* |
```

Comment:
1. The latest update should be today's date?
2. Can we remove this? "Users should no longer experience issues accessing the applications." -- users didn't experience issues accessing application as far as I know